cmd/thor/main.go

```diff
@@ -170,6 +174,16 @@ func defaultAction(ctx *cli.Context) error {
 		defer func() { log.Info("stopping metrics server..."); close() }()
 	}
 
+	adminURL := ""
+	if ctx.Bool(enableAdminFlag.Name) {
+		url, close, err := startAdminServer(ctx.String(adminAddrFlag.Name), logLevel)
```

Comment:
Do we have a default address in case one is not provided? (Same pattern as metrics)

Reply:
additionally can we check if it does not hijack websockets :D (not sure where to put it)

Reply:
@darrenvechain The default address is `http://localhost:2113` which is defined in the cli arguments if I am not mistaken

Reply:
@leszek-vechain What do you mean by "hijack websockets"? 🤔

Reply:
when Pedro did add the server for metrics I think, websockets were redirected there if I correctly remember, so just trying to avoid it happening again

Reply:
I am not familiar with that happened there. I just created a new server on a different port. Hopefully it should not affect anything else.